src/build.rs

```diff
@@ -0,0 +1,112 @@
+use reqwest;
+
+const URL: &str = "https://github.com/zksecurity/starknet-adapter/releases/download/v0.1.0-alpha/starknet-adapter-cli-linux-x86-64.tar.gz";
+const FILENAMES: [&str; 3] = ["cairo1-run", "cpu_air_prover", "cpu_air_verifier"];
```

Comment:
I think it would be a good idea to authenticate the downloads against hash digests:

```rust
const FILENAMES: [(&str, &str); 3] = [
     ("cairo1-run", "ab13cded..."),
     ...
];
```

Reply:
Done